native/mod_manager/mod_manager.c

```diff
@@ -3719,18 +3719,27 @@ static const char *cmd_manager_maxmesssize(cmd_parms *cmd, void *mconfig, const
     return NULL;
 }
 
-static const char *cmd_manager_enable_mcpm_receive(cmd_parms *cmd, void *dummy)
+static const char *cmd_manager_enable_mcmp_receive(cmd_parms *cmd, void *dummy)
 {
     mod_manager_config *mconf = ap_get_module_config(cmd->server->module_config, &manager_module);
     (void)dummy;
 
     if (!cmd->server->is_virtual) {
-        return "EnableMCPMReceive must be in a VirtualHost";
+        return "EnableMCMPReceive must be in a VirtualHost";
     }
-    mconf->enable_mcpm_receive = 1;
+    mconf->enable_mcmp_receive = 1;
     return NULL;
 }
 
+static const char *cmd_manager_enable_mcmp_receive_deprecated(cmd_parms *cmd, void *dummy)
```

Comment:
Comments from February? :D How did you do that? 

> Actually we should deprecate in the 1.3.x no? and that the correct one in 2.x

We really need to keep this in for compatibility and upgradeability... now when people use the legacy option they will get a warning and have an opportunity to fix their config otherwise people are going to copy their configs and would have no idea why they are failing.

Reply:
I would continue with the backporting and after a while (2.0.Final, 2.1.Final) we may remove the misspelled variant).

Reply:
> I would continue with the backporting and after a while (2.0.Final, 2.1.Final) we may remove the misspelled variant).

Correct - that's also the part of the idea of doing it this way, so it can be backported 1:1 since we definitely need to keep compatibility in the 1.3.x version.